package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "soap",
-  "version": "0.38.0",
+  "version": "0.39.0",
```

Comment:
please revert this version bump.  we'll take care of that when we publish.

Reply:
done

Reply:
Please tell me this isn't going to  take another 2 weeks to approve :)

Reply:
Sigh, looks like have to wait till at least 0.40.0